agentic/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@editor-extensions/agentic",
+  "version": "0.1.0",
+  "private": true,
+  "type": "module",
+  "types": "./dist/index.d.ts",
+  "main": "./dist/index.js",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.js",
+      "default": "./dist/index.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "scripts": {
+    "clean": "rimraf dist",
+    "ci": "npx jest",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix .",
+    "prebuild": "npm run clean",
+    "build": "npm run prebuild && tsup src/index.ts --dts --format esm --out-dir dist --clean",
+    "dev": "tsup src/index.ts --dts --format esm --out-dir dist --watch"
```

Comment:
why not just reuse the vite config from shared instead of having another build tool?

Reply:
I am not very familiar with the build tooling, when i tried to use vite config, it didnt work because its building it for webviews which dont have filesystem apis needed by langgraph. So i chose the path of least resistance right now, but if you guys can help me fix those issues, happy to learn.

Reply:
@sjd78 can you help me with above?

Reply:
I don't think anything in /agentic needs to be bundled for the browser, right?